src/api/EmployerFavouritesApi/Startup.cs

```diff
@@ -27,12 +33,21 @@ public Startup(IConfiguration configuration, IHostingEnvironment env)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddADAuthentication(Configuration, Environment);
+            services.AddMediatR(typeof(SaveApprenticeshipFavouriteCommand).Assembly);
+
             services.AddMvc(options => {
                 if (!Environment.IsDevelopment())
                 {
                     options.Filters.Add(new AuthorizeFilter("default"));
                 }
             }).SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+
+            services.AddSingleton<IAccountApiConfiguration>(x => Configuration.GetSection("AccountApiConfiguration").Get<AccountApiConfiguration>());
```

Comment:
Is the AccountApi configuration required?

Reply:
this is for the MA employer account api so i believe it is